test/re/frida/MethodTest.java

```diff
@@ -54,6 +58,131 @@ public void replacementPropagatesExceptions() {
         thrown.expectMessage("Already dead");
         badger.die();
     }
+    
+    @Test
+    public void interfaceCannotBeInstantiated() {
+        loadScript("var X509TrustManager = Java.use('javax.net.ssl.X509TrustManager');" +
+                "try {" +
+                "  var tm = X509TrustManager.$new();" +
+                "} catch (e) {" + 
+                "  send('couldnt create trustmanager');" + 
+                "}" + 
+                "send('ok');"
```

Comment:
Shouldn't this be right after `var tm = ...`, and the assert below should check against the error-case? (As the test is checking that it is invalid to instantiate an interface.) We should however have another test that illustrates how to register a new class implementing an interface in pure JS – or perhaps this test is meant to evolve into that?